pyqmc/orbitals.py

```diff
@@ -333,7 +330,17 @@ def aos(self, eval_str, configs, mask=None):
                 Ls=self.Ls,
             )
         )
-        ao = gpu.cp.einsum("k...,k...a->k...a", wrap_phase, ao)
+        # If gamma-point only, do not compute wrap_phase, keep AOs real-valued
```

Comment:
I still don't quite understand this. Why is this sum being evaluated every time? 
If we are returning real values, is self.complex still True? 

The decision should be made in __init__ not in aos().

Also I'm a little worried about summing self._kpts(). Wouldn't that be zero also if we have an even mesh? This is not the same as checking whether we are gamma. Do we also know that the AOs are always real for gamma? What about when there is spin-orbit?